neuro_san/internals/chat/data_driven_chat_session.py

```diff
@@ -57,7 +61,7 @@
 
 
 # pylint: disable=too-many-instance-attributes
-class DataDrivenChatSession:
+class DataDrivenChatSession(RunTarget):
```

Comment:
Make this guy derive from the new RunTarget interface.
This allows everything inside its new run_it() method be traceable.